app/src/main/java/io/branch/branchster/SplashActivity.java

```diff
@@ -58,10 +59,16 @@ public void onInitFinished(BranchUniversalObject branchUniversalObject, LinkProp
                 if (branchUniversalObject == null) {
                     proceedToAppTransparent();
                 }
+                /* In case the clicked link has $android_deeplink_path the Branch will launch the MonsterViewer automatically since AutoDeeplinking feature is enabled.
+                 * Launch Monster viewer activity if a link clicked without $android_deeplink_path
+                 */
                 else if (!branchUniversalObject.getMetadata().containsKey("$android_deeplink_path")) {
```

Comment:
But the auto deep link key is for a deep link path: `<meta-data android:name="io.branch.sdk.auto_link_path" android:value="monster/view/" />`. Just to make it clear, marketing links and iOS created links don't have deep link path. Here's one to try with: bnc.lt/ALMc/o6F0V0xU7u